bridge-impl/src/main/java/com/liferay/faces/bridge/context/internal/BridgeContextImpl.java

```diff
@@ -294,13 +294,6 @@ public BridgeURL encodeRedirectURL(String baseUrl, Map<String, List<String>> par
 
 		FacesContext facesContext = FacesContext.getCurrentInstance();
 
-		if (isJSF2PartialRequest(facesContext)) {
-
-			ArrayList<String> bridgeAjaxRedirect = new ArrayList<String>();
-			bridgeAjaxRedirect.add("true");
-			parameters.put("_bridgeAjaxRedirect", bridgeAjaxRedirect);
```

Comment:
@ngriffin7a,
Line 301 causes the `NullPointerException` mentioned in [FACES-2485](https://issues.liferay.com/browse/FACES-2485).